apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -195,6 +195,7 @@ window.CleverTakeoverManager = function (options) {
 
   function onCancelModal() {
     $("#user_user_type").val("student");
+    $.get("/users/clever_modal_dismissed");
```

Comment:
Bug 4

Reply:
Question: Why `GET` instead of `POST` here?  I can see arguments either way, and I'm curious what you had in mind.

My first instinct is `POST` because we're changing state, and we don't necessarily want this route to be cached, bookmarked, or navigated to manually.  But when I [went looking for guidelines](http://blog.teamtreehouse.com/the-definitive-guide-to-get-vs-post), `GET` also seems reasonable here because the operation is idempotent and therefore "safe," and apparently GET is preferred for AJAX because it's a faster operation (news to me).

Second question: should we ensure this request is successful, or are we cool with fire-and-forget here?  What's the negative user impact if we return a 500 here?

Reply:
I'm not really sure re: GET vs POST, since it is changing state it actually may make more sense for it to be a POST. I think because no data goes along with it I picked GET (also I wanted to test the method manually in the browser), but I may actually change this because you're right, POST makes more sense re: caching and whatnot.

The user impact if this fails is pretty minimal, it just means that the dialog will show up again if they navigate back to the home page during the same session. That's a minor annoyance, so I opted to ignore errors rather than surfacing or retrying.